azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecServerModule.java

```diff
@@ -30,11 +37,49 @@
  */
 public class AzkabanExecServerModule extends AbstractModule {
 
+  private static final Logger logger = Logger.getLogger(AzkabanExecServerModule.class);
+  private static final String EVENT_REPORTER_CLASS_PARAM = "event.reporter.class";
```

Comment:
Please define all configuration parameters in the azkaban.Constants class. This makes it easier to identify all the configuration keys.

Reply:
Sure, will do.

Reply:
done.